.pre-commit-config.yaml

```diff
@@ -7,28 +7,28 @@ repos:
         name: black
         language: system
         types: [python]
-        entry: command -v task > /dev/null && CMD=task || CMD=go-task; ${CMD} lint:black --
+        entry: bash -c 'command -v task > /dev/null && CMD=task || CMD=go-task; ${CMD} lint:black --'
```

Comment:
you need to pass the filenames in each entry
`bash -c 'command -v task > /dev/null && CMD=task || CMD=go-task; ${CMD} lint:black -- "$@"' --`

(except migrations, where we are not passing files)

Reply:
fixed